src/react-components/src/__tests__/components/Breadcrumbs-test.tsx

```diff
@@ -27,4 +27,15 @@ describe("Breadcrumbs", () => {
     let links = wrapper.render();
     expect(links.find("a.breadcrumbs__link").first().text()).to.equal("Hello");
   });
+
+  it("Renders icon", () => {
+    wrapper = Enzyme.shallow(<Breadcrumbs breadcrumbs={breadcrumbComponent} />);
+    let links = wrapper.render();
+
+    expect(links.find("svg")).to.have.lengthOf(1);
+  });
+  it("Throws error when nothing is passed into Breadcrumb", () => {
+    expect(() => Enzyme.mount(<Breadcrumbs breadcrumbs={[]} />))
+    .to.throw("Breadcrumbs cannot be empty");
```

Comment:
oh gosh, you and me both.  This is better though.  
Breaadcrumbs should always have svg, it's just whether or not the hidden styling is applied in mobile.  Since the hiding is done in the CSS, it doesn't make sense to test it here.